drivers/core_demographics_report/app/models/core_demographics_report/age_calculations.rb

```diff
@@ -152,25 +174,47 @@ def age_data_for_export(rows)
         rows["_#{age_category_title} Break"] ||= []
         rows["*#{age_category_title}"] ||= []
         rows["*#{age_category_title}"] += ['Gender', nil, 'Count', 'Average Age', nil]
+        available_coc_codes.each do |coc_code|
+          rows["*#{age_category_title}"] += [coc_code]
+        end
```

Comment:
Isn't this the same as 
```suggestion
        rows["*#{age_category_title}"] += available_coc_codes
```